apps/storybook/.storybook/preview.ts

```diff
@@ -2,38 +2,45 @@ import 'normalize.css';
 import '@h5web/shared/styles.css'; // importing via `@h5web/lib/src/styles.css` fails due ot lack of `~` prefix (as it is not supported by Vite)
 import '../src/styles.css';
 
-export const parameters = {
-  actions: { argTypesRegex: '^on[A-Z].*' },
-  docs: { source: { excludeDecorators: true } },
-  options: {
-    storySort: {
-      order: [
-        'Getting started',
-        'Utilities',
-        'Context',
-        'Customization',
-        'Visualizations',
-        ['LineVis'],
-        'Building Blocks',
-        [
-          'VisCanvas',
-          'Interactions',
+import type { Preview } from '@storybook/react';
+
+const preview: Preview = {
+  tags: ['autodocs'],
+  parameters: {
```

Comment:
No changes to this object. Just following the syntax suggested in Storybook's documentation, which actually bring better auto-completion thanks to the `Preview` type.